python/tk_hiero_export/sg_shot_processor.py

```diff
@@ -70,14 +79,18 @@ def populateUI(self, widget, exportItems, editMode=None):
 
         # create some helpful text
         header_text = QtGui.QLabel()
-        header_text.setText("""<big>Welcome to the Shotgun Shot Export!</big>
-                      <p>When you are using the Shotgun Shot Processor, Shots and Sequences in<br>
-                      Shotgun will be created based on your Hiero Project. Existing Shots will<br>
-                      be updated with the latest cut lengths. Quicktimes for each shot will be <br>
-                      sent to Screening Room for review when you use the special Shotgun <br>
-                      Transcode plugin - all included and ready to go in the default preset.<br>&nbsp;
-                      </p>
-                      """)
+        header_text.setWordWrap(True)
+        header_text.setText(
+            """
+            <big>Welcome to the Shotgun Shot Export!</big>
+            <p>When you are using the Shotgun Shot Processor, Shots and Sequences in
+            Shotgun will be created based on your Hiero Project. Existing Shots will
+            be updated with the latest cut lengths. Quicktimes for each shot will be
+            sent to Screening Room for review when you use the special Shotgun
```

Comment:
Totally meant to address these yesterday before pushing... I'll update these and get a new version to the app store.